packages/dds/tree/src/shared-tree/treeAlpha.ts

```diff
@@ -481,6 +513,72 @@ export const TreeAlpha: TreeAlpha = {
 		const parentSchema = treeNodeApi.schema(parent);
 		return getPropertyKeyFromStoredKey(parentSchema, storedKey);
 	},
+
+	child: (node: TreeNode, key: string | number): TreeNode | TreeLeafValue | undefined => {
+		const flexNode = getOrCreateInnerNode(node);
+		debugAssert(() => !flexNode.context.isDisposed() || "FlexTreeNode is disposed");
```

Comment:
since we must ensure this never fires with input a user could provide, it would be good to test that a proper UsageError is thrown when given a disposed node.

Reply:
Presumably the way to repro this would be to dispose the containing view before calling this?